admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/server/dao/provider/metadata/MapFieldsFieldMetadataProvider.java

```diff
@@ -171,7 +170,6 @@ public MetadataProviderResponse addMetadataFromFieldType(AddMetadataFromFieldTyp
                         ((BasicFieldMetadata) entry.getValue()).getExplicitFieldType(),
                         myType.getReturnedClass(), addMetadataFromFieldTypeRequest.getDynamicEntityDao()), metadata);
 
-                TypeConfigurationRegistry.INSTANCE.deregisterTypeConfiguration(typeConfiguration);
```

Comment:
is this class missing ? or why did you delete it ?

Reply:
It is missing, because Hibernate was upgraded to 5.6.11